dex/testing/dcr/harness.sh

```diff
@@ -217,23 +248,36 @@ for WALLET in alpha beta trading1 trading2; do
   tmux send-keys -t $SESSION:0 "./${WALLET} getnewaddress${WAIT}" C-m\; wait-for donedcr
 done
 
-echo "Mining 160 blocks on alpha"
-tmux send-keys -t $SESSION:0 "./mine-alpha 160${WAIT}" C-m\; wait-for donedcr
-
-sleep 5
-
-# Have alpha send some credits to the other wallets
-for i in 10 18 5 7 1 15 3 25
-do
-  RECIPIENTS="{\"${BETA_MINING_ADDR}\":${i},\"${TRADING_WALLET1_ADDRESS}\":${i},\"${TRADING_WALLET2_ADDRESS}\":${i}}"
-  tmux send-keys -t $SESSION:0 "./alpha sendmany default '${RECIPIENTS}'${WAIT}" C-m\; wait-for donedcr
-done
-sleep 0.5
-tmux send-keys -t $SESSION:0 "./mine-alpha 1${WAIT}" C-m\; wait-for donedcr
-
-# Create fee account on alpha wallet for use by dcrdex simnet instances.
-tmux send-keys -t $SESSION:0 "./alpha createnewaccount server_fees${WAIT}" C-m\; wait-for donedcr
-tmux send-keys -t $SESSION:0 "./alpha getmasterpubkey server_fees${WAIT}" C-m\; wait-for donedcr
+if [ "$MINE" = "1" ]; then
+  echo "Mining 600 blocks on alpha"
+  echo "Mining blocks 0 through 99"
+  tmux send-keys -t $SESSION:0 "./mine-alpha 100${WAIT}" C-m\; wait-for donedcr
+
+  # Send beta some dough while we're here.
+  tmux send-keys -t $SESSION:0 "./alpha sendtoaddress ${BETA_MINING_ADDR} 1000${WAIT}" C-m\; wait-for donedcr
+
+  echo "Mining blocks 100 through 199"
+  tmux send-keys -t $SESSION:0 "./mine-alpha 100${WAIT}" C-m\; wait-for donedcr
+  echo "Mining blocks 200 through 299"
+  tmux send-keys -t $SESSION:0 "./mine-alpha 100${WAIT}" C-m\; wait-for donedcr
+  echo "Mining blocks 300 through 399"
+  tmux send-keys -t $SESSION:0 "./mine-alpha 100${WAIT}" C-m\; wait-for donedcr
+  echo "Mining blocks 400 through 499"
+  tmux send-keys -t $SESSION:0 "./mine-alpha 100${WAIT}" C-m\; wait-for donedcr
+  # Don't stop here. There's a period of high ticket price where the avaialable
+  # balance for alpha is really low. Go to 600 to get through it.
+  echo "Mining blocks 500 through 599"
```

Comment:
Another thought, perhaps balancetomaintainabsolute is too high. https://github.com/decred/dcrdex/pull/820/files#diff-9698effccb6bff89806e316b1ed2c16f269173e49dd87aeae51fe24a3b4c0317R45

Reply:
Would that affect the actual ticket price though? The ticket price at block 500 is like 2,100 DCR.

Reply:
No I was just thinking if it was running out of funds for tickets maybe it could hold less in reserve.  It's fine with the archive I'm sure.